cllc-public-app/ClientApp/src/app/components/applications/application/tabs/lg-in-confirmation-of-receipt/lg-in-confirmation-of-receipt.component.html

```diff
@@ -68,25 +72,25 @@ <h3 class="blue-header">
 </section>
 
 <section class="position-relative d-flex justify-content-between p-3">
-    <span>
-        <button mat-raised-button color="secondary" [disabled]="optingOutOfComment" (click)="OptOutOfComment()">
-            <span>Opt Out of Comment</span>
-            <mat-progress-bar *ngIf="optingOutOfComment" mode="indeterminate"></mat-progress-bar>
-        </button>
-    </span>
+    <button mat-raised-button color="secondary" [disabled]="optingOutOfComment" (click)="OptOutOfComment()">
+        <span>Opt Out of Comment</span>
+        <mat-progress-bar *ngIf="optingOutOfComment" mode="indeterminate"></mat-progress-bar>
+    </button>
 
-    <span>
-        <button mat-raised-button color="secondary" [disabled]="rejectingApp" (click)="RejectApplication()">
-            <span>Reject Application</span>
-            <mat-progress-bar *ngIf="rejectingApp" mode="indeterminate"></mat-progress-bar>
-        </button>
-    </span>
+    <button mat-raised-button color="secondary" [disabled]="rejectingApp" (click)="RejectApplication()">
+        <span>Reject Application</span>
+        <mat-progress-bar *ngIf="rejectingApp" mode="indeterminate"></mat-progress-bar>
+    </button>
 
-    <span>
-        <button mat-raised-button color="primary" [disabled]="approvingApplication" (click)="ApproveApplication()">
-            <span>Accept Application</span>
-            <mat-progress-bar *ngIf="approvingApplication" mode="indeterminate"></mat-progress-bar>
-        </button>
-    </span>
+    <button mat-raised-button color="primary" *ngIf="application?.lGApprovalDecision  !== 'Pending'"
+        [disabled]="approvingApplication" (click)="ApproveApplication()">
+        <span>Accept Application</span>
+        <mat-progress-bar *ngIf="approvingApplication" mode="indeterminate"></mat-progress-bar>
+    </button>
 
-</section>
+    <button mat-raised-button color="primary" *ngIf="application?.lGApprovalDecision  === 'Pending'"
+        [disabled]="approvingApplication" (click)="ProvideResolution()">
```

Comment:
Question: shouldn't this be disabled when "providingResolution" is true instead of approvingApplication?